### ISSUE TITLE
Add repo to manage Helm charts for kubernetes-sigs

### ISSUE THREAD

**Issue by stevehipwell** [kubernetes/community#5712]
It would be really useful to have a single repository under _kubernetes-sigs_ to manage Helm charts in one place for the projects hosted there. This repo could be configured to use GitHub actions to lint and test the charts before releasing them to a Helm repo hosted on the GitHub pages branch. I'd suggest `kubernetes-sigs/helm-charts` would be a good repo for this and I'd be keen to collaborate on it.

A Helm charts repo would allow projects to have a Helm chart without needing to deal with the lifecycle events within their repo, which seems to be a blocker for some projects. It would also allow projects which already have a chart to move it to a central location with minimal effort.

End-users would have a simpler task of finding the correct charts as they would only need to add a single Helm repo.

**Comment by mrbobbytables** [comment]
Just as an FYI - this issue likely won't get attention from other groups here. You'll reach folk by sending it to the SIG mailing list or kubernetes-dev directly.

Would this solve your problem? - https://github.com/kubernetes/org/issues/2587
TL;DR an artifact-hub org account for k-sigs where they could be published there.

I'm hesitant on a mono-repo for charts as it'd constantly be a secondary thing to update vs being managed alongside the parent project. It caused a lot of issues for Helm itself and was a big reason why they moved away from the singular helm-charts repo.

**Comment by stevehipwell** [comment]
@mrbobbytables thanks for your response. I'm unfamiliar with how this process works so if you could give me a bit more of a pointer to move this forward that would help.

I think kubernetes/org#2587 would be useful for the repos where they've already added a chart but need to get it registered in https://artifacthub.io/. I don't think it addresses or solves the problem of projects not wanting the Helm chart in their own repo.

Personally I like the chart in the application repo pattern as the chart is just a manifestation of the yaml which is already being maintained (a chart maintainer would just be supporting any yaml changes or general chart improvements). I've had success getting this implemented in a couple of repos but it's being blocked on other important repos ([metrics-server](https://github.com/kubernetes-sigs/metrics-server/pull/670) & [external-dns](https://github.com/kubernetes-sigs/external-dns/issues/1941) specifically).

A central repo would act as a catch-all for charts which should exist but don't currently and unless there is a way to require maintainers to allow charts this should be a priority. The discussion around the correct location for charts is a separate issue and would be better had once the charts in question actually exist.

**Comment by mrbobbytables** [comment]
I would move forward with mailing the sig-apps mailing list. All repos require a sponsoring SIG to adopt and take ownership of the management of the repo.

**Comment by stevehipwell** [comment]
@mrbobbytables I've put this on the [sig-apps mailing list](https://groups.google.com/g/kubernetes-sig-apps/c/FB4WgACGQoo).

**Comment by pierluigilenoci** [comment]
@stevehipwell we need to find someone to ping 😜

**Comment by stevehipwell** [comment]
@pierluigilenoci this issue might be obsolete now that projects have added charts to their repos?

**Comment by pierluigilenoci** [comment]
Make sense
